ui/app/utils/openapi-to-attrs.js

```diff
@@ -30,6 +30,23 @@ export const expandOpenApiProps = function (props) {
       editType = items.type + capitalize(type);
     }
 
+    // remove references to comma separated strings from tooltips
+    if (editType === 'stringArray') {
+      if (description?.includes('Comma separated string or')) {
+        description = description.replace(/\bComma\b \bseparated\b \bstring\b \bor\b |\bJSON\b /g, '');
+        description = description.replace(/\blist\b|\barray\b/, 'List');
+      } else if (description?.includes('Comma separated list')) {
+        description = description.replace(/\bComma\b \bseparated\b \blist\b/, 'List');
+      } else if (description?.includes('Comma-separated list')) {
+        description = description.replace(/\bComma\b-\bseparated\b \blist\b/, 'List');
+      } else if (description?.includes('comma-separated list')) {
+        description = description.replace(/\bcomma\b-\bseparated\b /, '');
+      } else if (description?.includes('comma-separated string or')) {
+        description = description.replace(/\bcomma\b-\bseparated\b \bstring\b \bor\b /, '');
+        description = description.replace(/\barray\b/, 'list');
+      }
+    }
+
```

Comment:
I wonder if this is something we could work with the backend folks on to get the descriptions updated there instead?

Reply:
Messaged in slack, but here for posterity. Since these are used for the CLI, my understanding is we can't change them which is why I proposed intercepting them here. 

We definitely don't have to go this route! It was just a thought since this has caused ongoing UX headaches for users. 

Ideally we'd intercept them in the model for the corresponding params (like we do for PKI), but I couldn't find a way to do this with how the models are currently generated for auth-methods and their resources (i.e. roles)